go.mod

```diff
@@ -15,10 +15,11 @@ require (
 	github.com/hashicorp/go-uuid v1.0.1 // indirect
 	github.com/hashicorp/logutils v1.0.0
 	github.com/hashicorp/mdns v1.0.1
-	github.com/hashicorp/memberlist v0.2.2
+	github.com/hashicorp/memberlist v0.3.2-0.20220729184846-d8175c29c16b
```

Comment:
After the changes to memberlist are merged we should git tag the special backport branch (NOT with semver) and then update that here. Pointing directly to a git sha from go.mod has backfired for folks in the past for long term tracking.

Reply:
Hi, @rboyer , I tried the following command to add the special backport branch

```
# go get -d  github.com/hashicorp/memberlist@metrics-labels-backport
go: added github.com/hashicorp/memberlist v0.3.2-0.20220801164105-62e12a6f0768
```

It turns out the version added to go.mod is still using the git sha. Any suggestions? Thanks.

Reply:
I figured it out that I missed tagging the version in memberlist. The go.mod has been updated.